src/pipeline.ml

```diff
@@ -282,25 +298,16 @@ module Tarides = struct
       ];
     ]
 
-  let docker ~sched ~staging_auth t =
-    let timeout =
-      if List.mem `Linux_riscv64 t.archs then Int64.mul Build.timeout 2L
-      else Build.timeout
-    in
-    let sched = Current_ocluster.v ~timeout ?push_auth:staging_auth sched in
-    docker ~sched t
```

Comment:
All this logic is moved into the previous `docker` function, so we can avoid the convoluting shadowing and reuse the same logic more consistently.